Scripts/Miscellaneous/Unix_to_Human/README.md

```diff
@@ -0,0 +1,21 @@
+# Python script that converts UNIX timestamp to huam interpretable format
+A simple Python script that uses 'time(inbuilt)' module to convert timestamp to data and time format.
+
+### Prerequisites
+```time``` Module
+* The time module is inbuilt with Python. 
+
+### How to run the script
+First you have to go to the Unix_to_Human directory. Run the following command once you are in project directory
+
+```cd Scripts/Miscellaneous/Unix_to_Human```
+
+For Python 3: ```python3 wifiSpeed.py```
```

Comment:
execution file name is different than your pushed code
```suggestion
For Python 3: ```python3 timestamp_to_human.py```
```